decoration.sk

```diff
@@ -122,8 +122,22 @@ decoratives before flattening:
 	unlit furnace¦s = minecraft:furnace
 	lit furnace¦s = minecraft:lit_furnace
 	[any] furnace¦s = unlit furnace, lit furnace
-	ladder¦s = minecraft:ladder
+	
+	ladder item¦s = minecraft:ladder
+	south[ward] facing ladder = minecraft:ladder {Damage:0}
+	north[ward] facing ladder = minecraft:ladder {Damage:3}
+	east[ward] facing ladder = minecraft:ladder {Damage:4}
+	west[ward] facing ladder = minecraft:ladder {Damage:5}
+	
 	snow layer¦s = minecraft:snow_layer
```

Comment:
How about adding a `(1|one) layer snow layer = minecraft:snow_layer {Damage:0}` here? That way they can specifically target a thin snow layer.

Additionally, I don't love the alias name here; "two layer snow layer" seems a bit repetitive. Maybe change it to "thick", such as `two thick snow layer`? Open to opinions.